certified-connectors/Kanbanize/apiDefinition.swagger.json

```diff
@@ -0,0 +1,910 @@
+                    "parameters": {
+                      "board_id": {
+                        "parameter": "boardid"
+                      }
+                    }
+                  },
+                  "x-ms-dynamic-list": {
+                    "operationId": "GetLanes",
+                    "itemsPath": "data",
+                    "itemValuePath": "name",
+                    "itemTitlePath": "name",
+                    "parameters": {
+                      "board_id": {
+                        "parameterReference": "body/boardid"
+                      }
+                    }
+                  }
+                },
+                "position": {
+                  "type": "string",
+                  "description": "position"
+                },
+                "exceedingreason": {
+                  "type": "string",
+                  "description": "exceedingreason"
+                }
+              },
+              "required": [
+                "boardid"
+              ]
+            }
+          }
+        ]
+      }
+    },
+    "/index.php/api/kanbanize/add_subtask/format/json": {
+      "post": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "summary": "Create Subtask",
+        "description": "Creates a new subtask of a card or initiative",
+        "operationId": "CreateSubtask",
+        "parameters": [
+          {
+            "name": "Content-Type",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "application/json",
+            "x-ms-visibility": "advanced",
+            "description": "Always use: application/json",
+            "x-ms-summary": "Content Type"
+          },
+          {
+            "name": "Accept",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "application/json",
+            "x-ms-visibility": "advanced",
+            "description": "Always use: application/json",
+            "x-ms-summary": "Content Type"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": false,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "taskparent": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "taskparent"
+                },
+                "title": {
+                  "type": "string",
+                  "description": "title"
+                },
+                "assignee": {
+                  "type": "string",
+                  "description": "assignee"
+                }
+              }
+            }
+          }
+        ]
+      }
+    },
+    "/index.php/api/kanbanize/add_comment/format/json": {
+      "post": {
+        "responses": {
+          "default": {
```

Comment:
@dimitar-kanbanize Please replace the "default" responses with expected status codes, otherwise users may see errors about incorrect schema instead of real error responses from your service in OpenAPI mode.